portfolio/src/main/webapp/index.html

```diff
@@ -36,31 +36,31 @@
       <div class="color-buttons">
         <li class="nav-item" id="paint">
           <a class="nav-link white">
-            <i class="material-icons">format_paint</i>
+            <i class="material-icons" style="color: var(--text-primary)"> format_paint</i>
```

Comment:
I really like how you simplified this code! It's not just that there's less code, but also, it's so tidy that the actual color of the button itself controls the color of everything else - it prevents any possible mistakes like mixing up which button is assigned to which color.